sandboxes/horizontal-bar/src/components/horizontalBarChart.vue

```diff
@@ -5,6 +5,16 @@
 
 <script>
 import { HorizontalBar } from 'vue-chartjs'
```

Comment:
I think that the `horizontalBar` can be removed, because its not a custom type anymore, but part of the normal bar chart: https://www.chartjs.org/docs/latest/samples/bar/horizontal.html

Reply:
removed )